support/azure/active-directory/azure-ad-ipv6-support.md

```diff
@@ -65,6 +65,68 @@ For the IP ranges that are specified for Azure AD, make sure that you allow outb
 
 By default, both IPv6 and IPv4 traffic is supported on Windows and most other operating system (OS) platforms. Changes to the standard IPv6 configuration may result in unintended consequences. For more information, see [Guidance for configuring IPv6 in Windows for advanced users](/troubleshoot/windows-server/networking/configure-ipv6-in-windows).
 
+## How to find the IPv6 address used to sign-in to Azure AD
+To help admins find the IPv6 addresses used to sign-in to Azure AD, either during proactive testing 
+[Name Resolution Policy Table (NRPT)](https://learn.microsoft.com/en-us/troubleshoot/azure/active-directory/azure-ad-ipv6-support#test-azure-ad-authentication-over-ipv6) mentioned above OR post the IPv6 enablement in March 2023, use the methods below. Finding the top IPv6 addresses that are owned and managed by the organization will help you add the same under ‘Named location”.
+
+### Methods to get the list of IPv6 addresses from Sign-in logs:
+
+   #### Filtering Azure AD Sign-in logs
+   - In the Azure portal, search for and select Azure Active Directory.
```

Comment:
What role is needed?

Reply:
To access the sign-ins log for a tenant, you must have one of the following roles:

Global Administrator
Security Administrator
Security Reader
Global Reader
Reports Reader